dds_web/templates/mail/project_release.html

```diff
@@ -18,22 +18,29 @@
         </ul>
       </p>
       <p style="font-family: arial, sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">
+        You were added to this project on behalf of {{displayed_sender}}.
```

Comment:
We don't want to print "on behalf of" if the user giving access to the project was a project owner for example. In this case, the unit_email will be none, so we need to have that check still. 
```suggestion
        You were added to this project {% if unit_email %} on behalf of {% else %} by {% endif %} {{displayed_sender}}.
```